src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -94,7 +94,7 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         model.setPerson(personToEdit, editedPerson);
-        logger.info("Edit command has been exceuted successfully on a person with NRIC: " + predicate);
+        logger.info("Edit command has been executed successfully on a person with NRIC: " + predicate);
```

Comment:
Just a nit: Instead of printing `predicate` for the logger message, you could choose to print `editedPerson.getNric()` for the actual NRIC number, since printing `predicate` will result in the Object being printed instead